src/mappings/StabilityPool.ts

```diff
@@ -75,26 +75,12 @@ export function handleETHGainWithdrawn(event: ETHGainWithdrawn): void {
   }
 }
 
-export function handleRBTCGainWithdrawn(event: RBTCGainWithdrawn): void {
-  // Leave a non-null dummy value to signal to handleUserDepositChanged()
-  // that ETH gains have been withdrawn
-  let depositUpdate = swapTmpDepositUpdate(
-    BIGINT_ZERO,
+export function handleSOVPaidToDepositor(event: SOVPaidToDepositor): void {
+  let SOVDistributionEntity = new SOVDistribution(
     event.transaction.hash.toHexString()
```

Comment:
To make sure entity is not overwritten if multiple events received in same transaction:
```suggestion
    event.transaction.hash.toHexString() + '/' + event.logIndex.toString()
```